exonum/src/helpers/fabric/details.rs

```diff
@@ -38,6 +38,8 @@ use super::shared::{AbstractConfig, NodePublicConfig, SharedConfig, NodePrivateC
                     CommonConfigTemplate};
 use super::DEFAULT_EXONUM_LISTEN_PORT;
 
+const DATABASE_PATH: &str = "DATABASE_PATH";
```

Comment:
I think the explicit constant is redundant.

Reply:
Why not? It is used three times. 😄

Reply:
One reason for using constant: rust will statically check typo.